.env.template

```
@@ -27,3 +27,10 @@ DEFAULT_ADMIN_PASSWORD=
 
 # CORS Config
 WEBAPP_HOST=http://localhost:5173
+
+# Port to use in the express server
+API_PORT=
+
+# Rate limiter config
+RATE_WINDOW=
```

Deixar claro que é o intervalo em minutos.
```suggestion
RATE_WINDOW_MINUTES=
```